GoogleForCreators/web-stories-wp
# Fix displaying whiteout in Text edit mode

## Summary


- Adds whiteout to the part of the markup that changes its height when adding new lines.
- Also, adds whiteout only if the background mode is not Fill -- otherwise this might interfere with the true color of the background if there's opacity.

## Relevant Technical Choices



## To-do



## User-facing changes

For edit mode only!
Change in case of media that has opacity: media with opacity now doesn't have whiteout and displays the actual color. In case of media without opacity, there is no change.
Before:
![before](https://user-images.githubusercontent.com/3294597/84887563-36d91200-b096-11ea-8b75-54f84c88a8e4.gif)

After:
![after](https://user-images.githubusercontent.com/3294597/84887572-3a6c9900-b096-11ea-83df-dc7e62461cf3.gif)




## Testing Instructions
1. Add a text element (set the Page background to something red for visibility)
2. Go to edit mode
3. Add a new line
4. Verify the background whiteout is extending to the new lines, too


---



Fixes #1883
